sdk/attestation/azure-security-attestation/src/attestation_client.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/attestation/attestation_client.hpp"
+#include <shared_mutex>
+
+#include "azure/core/base64.hpp"
+#include "azure/core/http/policies/policy.hpp"
+#include "azure/core/internal/http/pipeline.hpp"
+#include "private/attestation_client_models_private.hpp"
+#include "private/attestation_client_private.hpp"
+#include "private/attestation_common_request.hpp"
+#include "private/attestation_deserializers_private.hpp"
+#include "private/package_version.hpp"
+
+#include <string>
+
+using namespace Azure::Security::Attestation;
+using namespace Azure::Security::Attestation::Models;
+using namespace Azure::Security::Attestation::_detail;
+using namespace Azure::Security::Attestation::Models::_detail;
+using namespace Azure::Core::Http;
+using namespace Azure::Core::Http::Policies;
+using namespace Azure::Core::Http::_internal;
+
+std::string AttestationClient::ClientVersion() const { return PackageVersion::ToString(); }
+
+AttestationClient::AttestationClient(
+    std::string const& endpoint,
+    std::shared_ptr<Core::Credentials::TokenCredential const> credential,
+    AttestationClientOptions options)
+    : m_endpoint(endpoint), m_tokenValidationOptions(options.TokenValidationOptions)
+{
+  std::vector<std::unique_ptr<HttpPolicy>> perRetrypolicies;
+  if (credential)
+  {
+    //      Azure::Core::Credentials::TokenRequestContext const tokenContext
```

Comment:
remove commented out code

Reply:
At some point in the near future, this code will become live - it's currently a placeholder for what I hope to build in the next set of work items.